templates/shrine/src/shrine-app.html

```diff
@@ -0,0 +1,375 @@
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../../../../polymer/polymer.html">
+<link rel="import" href="../../../../iron-flex-layout/iron-flex-layout.html">
+<link rel="import" href="../../../../iron-media-query/iron-media-query.html">
+<link rel="import" href="../../../../iron-ajax/iron-ajax.html">
+<link rel="import" href="../../../../paper-icon-button/paper-icon-button.html">
+<link rel="import" href="../../../../paper-icon-button/paper-icon-button.html">
+<link rel="import" href="../../../../paper-tabs/paper-tabs.html">
+<link rel="import" href="../../../../iron-icons/iron-icons.html">
+<link rel="import" href="../../../../iron-page-url/iron-page-url.html">
+<link rel="import" href="../../../app-layout.html">
+<link rel="import" href="../../../app-scroll-effects/app-scroll-effects.html">
+<link rel="import" href="shrine-list.html">
+<link rel="import" href="shrine-detail.html">
+
+<dom-module id="shrine-app">
+
+  <template>
+
+    <style>
+
+      app-header {
+        position: fixed;
+        background-color: rgba(255, 255, 255, 0.95);
+        top: 0;
+        left: 0;
+        right: 0;
+        --app-header-shadow: {
+          box-shadow: inset 0px 5px 6px -3px rgba(0, 0, 0, 0.2);
+          height: 10px;
+          bottom: -10px;
+        };
+      }
+
+      app-header:not([shadow]) {
+        border-bottom: 1px solid #ddd;
+      }
+
+      app-drawer {
+        --app-drawer-content-container: {
+          background-color: #f3f3f3;
+        };
+      }
+
+      #mainToolbar {
+        height: 110px;
+        align-items: center;
+        justify-content: center;
+      }
+
+      paper-icon-button {
+        color: #7c7c7c;
+        --paper-icon-button-ink-color: #31f0ef;
+      }
+
+      .navItem {
+        align-self: flex-start;
+        margin-top: 13px;
+        width: 88px;
+      }
+
+      .spacer {
+        flex: 1 1;
+        -webkit-flex: 1 1;
+        display: flex;
+        align-items: center;
+        justify-content: center;
```

Comment:
You don't need `-webkit` prefix for the other stuffs? e.g. `display: -webkit-flex`.... also you might want to just use `@apply(--layout-horizontal);`